packages/solid/components/FocusRing/FocusRing.stories.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2024 Comcast Cable Communications Management, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import type { Meta, StoryObj } from 'storybook-solidjs';
+import FocusRing from './FocusRing.jsx';
+
+type Story = StoryObj<typeof FocusRing>;
+
+/**
+ * Fancy ring to focus on Items.
+ *
+ * ## Usage
+ *
+ * Add a focus ring to tile or other items to provide a nice highlight
+ * and shadow effect
+ *
+ * ```js
+ * import { View } from '@lightningtv/solid';
+ * import { useFocusManager } from '@lightningtv/solid/primitives';
+ * import { FocusRing } from '@lightningtv/solid-ui';
+ * import { setFocusRing } from '@lightningtv/solid-ui/utils';
+ *
+ * const App = () => {
```

Comment:
is the plan to restore this in the future?

Reply:
No this is the "How to use docs." You insert the docs in the stories file now with the new use of autodocs instead of the mdx.  I remember we wanted to add this information for the focusRing since it has a specific set up so this is just an example of how to use it. 
![Screenshot 2024-08-20 at 8 35 32 AM](https://github.com/user-attachments/assets/07db1086-8191-4555-b033-9d1ffa06ed4e)